src/vs/workbench/parts/debug/electron-browser/repl.ts

```diff
@@ -375,3 +390,26 @@ CommonEditorRegistry.registerEditorCommand(new SuggestCommand({
 		primary: KeyCode.RightArrow
 	}
 }));
+
+@editorAction
+class ReplCopyAllAction extends EditorAction {
+
+	constructor() {
+		super({
+			id: 'repl.action.copyall',
+			label: nls.localize('actions.repl.copyall', "Debug Copy All"),
+			alias: 'Debug Copy All',
+			precondition: debug.CONTEXT_IN_DEBUG_REPL,
+			kbOpts: {
```

Comment:
I think you can just leave out the `kbOpts` and it should nicely compile, please try that